db/schema.rb

```diff
@@ -141,7 +152,6 @@
     t.string "workflow", limit: 20, comment: "Workflow used when processing the flowcell"
     t.index ["flowcell_barcode", "position", "tag_index"], name: "index_iseqflowcell__flowcell_barcode__position__tag_index"
     t.index ["id_flowcell_lims", "position", "tag_index", "id_lims"], name: "index_iseq_flowcell_id_flowcell_lims_position_tag_index_id_lims", unique: true
-    t.index ["id_flowcell_lims", "position", "tag_index"], name: "index_iseqflowcell__id_flowcell_lims__position__tag_index"
```

Comment:
What's the reason for the deletion here? It looks like this index does exist in prod, and I don't think we want to be deleting an index from iseq_flowcell.